lib/src/services/dio_logger.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:dio/dio.dart';
+
+class DioLogger {
+  static void onSend(String tag, RequestOptions options) {
+    // ignore: avoid_print
```

Comment:
Instead of adding this before every print statement, I recommend adding `// ignore_for_file: avoid_print` at the top of the file and it will ignore `avoid_print` rule in one line.